File: Semester 4/Systems for Design and Implementation/django_backend/A1/snippets/models/Person.py
from django.db import models
from snippets.models.CityBreak import CityBreak

class Person(models.Model):
    first_name=models.CharField(max_length=100)
    last_name=models.CharField(max_length=100)
    age=models.IntegerField()
    gender=models.CharField(max_length=100)
    email=models.CharField(max_length=100)
    cityBreak=models.ForeignKey(CityBreak,on_delete=models.CASCADE)

    def __str__(self):
        return self.first_name

File: Semester 4/Systems for Design and Implementation/react_frontend/src/models/Person.ts
import { CityBreak } from "./CityBreak";

export interface Person{
    id:number;
    first_name:string;
    last_name:string;
    age:number;
    gender:string;
    email:string;
    cityBreak_id:number;
    cityBreak:CityBreak;
}